data-serving/data-service/src/model/location.ts

```diff
@@ -1,21 +1,29 @@
 import mongoose from 'mongoose';
 
+export enum GeoResolution {
+    Point = 'Point', // Lat/long coordinates
+    Admin3 = 'Admin3', // ~City
+    Admin2 = 'Admin2', // ~County
+    Admin1 = 'Admin1', // ~Province
+    Admin0 = 'Admin0', // Country
```

Comment:
Country/Admin0 isn't really an administrative area concept, is it? Might be clearer to call it country, so that it aligns with the schema field name.

Reply:
+1 for country, I made another comment about this

Reply:
replaced admin0 with country